packages/analytics/analytics-chart/src/components/chart-plugins/DragSelectPlugin.ts

```diff
@@ -37,14 +37,19 @@ const dispatchEvent = (eventName: string, chart: Chart, pluginInstance: DragSele
   }))
 }
 
+export class DragSelectPlugin implements IDragSelectPlugin {
```

Comment:
Since these aren't being shared across packages, only used internally, I think it's a good use case for classes?